src/esplog/mod.rs

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use std::io::*;
+use std::str::from_utf8;
+use std::sync::{
+    atomic::{AtomicBool, Ordering},
+    Arc,
+};
+
+use byteorder::{LittleEndian, ReadBytesExt};
+
+use crate::esplog::mini_decompressor::{decompress_block, Quat, State};
+use crate::tags_impl::*;
+use crate::*;
+
+mod mini_decompressor;
+
+#[derive(Default)]
+pub struct EspLog {
+    pub model: Option<String>,

```

Comment:
Will there be some kind of name in the binary file? If not, this field can be removed completely

Reply:
This field is required here https://github.com/AdrianEddy/telemetry-parser/blob/master/src/lib.rs#L76

Reply:
right, my bad